client/__tests__/e2e/cellxgeneActions.ts

```diff
@@ -602,43 +602,49 @@ export async function expandGene(
 
 export async function requestGeneInfo(gene: string, page: Page): Promise<void> {
   await page.getByTestId(`get-info-${gene}`).click();
-  await expect(page.getByTestId(`${gene}:gene-info`)).toBeTruthy();
+}
+
+export async function requestCellTypeInfo(cell: string, page: Page) {
+  await page.getByTestId(`cell_type:category-expand`).click();
+  await page.getByTestId(`get-info-cell_type-${cell}`).click();
 }
 
 export async function assertInfoPanelExists(
-  gene: string,
+  id: string,
+  entity: string,
   page: Page
 ): Promise<void> {
-  await expect(page.getByTestId(`${gene}:gene-info`)).toBeTruthy();
-  await expect(page.getByTestId(`info-panel-header`)).toBeTruthy();
-  await expect(page.getByTestId(`min-info-panel`)).toBeTruthy();
+  await expect(page.getByTestId(`${id}:${entity}-info-wrapper`)).toBeVisible();
```

Comment:
`entity` is what I used throughout as the general var for either `gene` or `cell-type` - open to naming this something else